src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/takecard/TakeCard.kt

```diff
@@ -21,52 +19,71 @@ import tornadofx.*
 
 class TakeCard(val take: Take, player: IAudioPlayer) : AnchorPane() {
     val playedProperty = SimpleBooleanProperty(take.played)
-    val badge = stackpane {
-        addClass(TakeCardStyles.badge)
-        val icon = TakeCardStyles.badgeIcon()
-        icon.addClass(TakeCardStyles.iconStyle)
-        add(icon)
-        isVisible = !take.played
-    }
 
     var deleteButton: Button by singleAssign()
     var takeNumberLabel: Label by singleAssign()
     var timestampLabel: Label by singleAssign()
-    var simpleAudioPlayer: SimpleAudioPlayer by singleAssign()
+    var playButton: Button by singleAssign()
+    var editButton: Button by singleAssign()
+
 
+    var simpleAudioPlayer: SimpleAudioPlayer by singleAssign()
+    private val isAudioPlaying: SimpleBooleanProperty = SimpleBooleanProperty()
     init {
         importStylesheet<TakeCardStyles>()
-        setRightAnchor(badge, 0.0)
-        setTopAnchor(badge, 0.0)
         addClass(TakeCardStyles.defaultTakeCard)
         vbox {
             addClass(TakeCardStyles.content)
+            //the top bar of the take card
             hbox(10) {
-                vgrow = Priority.ALWAYS
-                alignment = Pos.CENTER_LEFT
-                takeNumberLabel = label("%02d".format(take.number))
-                takeNumberLabel.addClass(TakeCardStyles.takeNumberLabel)
-                timestampLabel = label(take.timestamp.toString())
-                timestampLabel.addClass(TakeCardStyles.timestampLabel)
+                style {
+                    maxHeight = 75.0.px
+                }
+                hbox(10.0) {
+                    hgrow = Priority.ALWAYS
+                    alignment = Pos.TOP_LEFT
+                    takeNumberLabel = label("Take " + "%02d".format(take.number), TakeCardStyles.draggingIcon())
```

Comment:
is the take number limited to two digits in the filename?

Reply:
I'm not sure. This is formatting that existed from before. i'll put in a check and do this formatting if its a single digit number, otherwise i'll leave it as is

Reply:
I suppose it's fine, and it wouldn't break on a larger number, I just wanted to make sure that wasn't a limitation we re-introduced from tR lol

Reply:
"Take" is a word that probably needs localized